tests/dummy/config/environment.js

```diff
@@ -11,10 +11,7 @@ module.exports = function (environment) {
         // Here you can enable experimental features on an ember canary build
         // e.g. 'with-controller': true
       },
-      EXTEND_PROTOTYPES: {
-        // Prevent Ember Data from overriding Date.parse.
-        Date: false,
-      },
+      EXTEND_PROTOTYPES: false,
```

Comment:
Normally, in v4.6, the `EXTEND_PROTOTYPES` object was removed completely, but since this `ember-freestyle` does not have the `ember-disable-prototype-extensions` addon installed as a development dependency, I decided to set it to `false`, instead of installing the addon. `false` will be the new default soon anyways.